### PR TITLE
wait for scanning to finish

### PR DESCRIPTION
**Pull request by nataly87s**
https://github.com/Soluto/tweek/pull/653#issuecomment-344277847

### CODE REVIEW

File: TweekApiSmokeTestDockerfile [hunk 1]
```diff
@@ -1,6 +1,8 @@
 FROM microsoft/aspnetcore-build:2.0.0
 
-RUN wget https://raw.githubusercontent.com/vishnubob/wait-for-it/8ed92e8cab83cfed76ff012ed4a36cef74b28096/wait-for-it.sh && \
+RUN apt-get update && \
+    apt-get install -y jq && \
```
Comment (omerlh) [inline TweekApiSmokeTestDockerfile]: there are some best practices like `--no-recomendment` (not remember the syntax) and clear cache to make image smaller, not sure if it important here

File: e2e/ui/utils/client-utils.js [hunk 2]
```diff
@@ -1,6 +1,8 @@
 const axios = require('axios');
 const nconf = require('nconf');
 
+const delay = timeout => new Promise(resolve => setTimeout(resolve, timeout));
```
Comment (omerlh) [inline e2e/ui/utils/client-utils.js]: why it is added?

Comment (nataly87s) [reply e2e/ui/utils/client-utils.js]: I just noticed there was an error message while running the tests, so I fixed it. 
It's just a small bug fix, not really related to zap or glue

**Comment by omerlh** [comment]
Really awesome :) Waiting for the blog post!

**Pull request merged**